sdk/healthinsights/azure-healthinsights-cancerprofiling/README.md

```diff
@@ -0,0 +1,255 @@
+                                    id="doc1",
+                                    content=DocumentContent(source_type=DocumentContentSourceType.INLINE,
+                                                            value=doc_content1),
+                                    clinical_type=ClinicalDocumentType.IMAGING,
+                                    language="en",
+                                    created_date_time=datetime.datetime(2021, 8, 15))
+
+doc_content2 = "Oncology Clinic " \
+                + "20.10.2021" \
+                + "Jane Doe 091175-8967" \
+                + "42-year-old healthy female who works as a nurse in the ER of this hospital. " \
+                + "First menstruation at 11 years old. First delivery- 27 years old. She has 3 children." \
+                + "Didn’t breastfeed. " \
+                + "Contraception- Mirena." \
+                + "Smoking- 10 pack years. " \
+                + "Mother- Belarusian. Father- Georgian. " \
+                + "About 3 months prior to admission, she stated she had SOB and was febrile. " \
+                + "She did a CXR as an outpatient which showed a finding in the base of the right lung- " \
+                + "possibly an infiltrate." \
+                + "She was treated with antibiotics with partial response. " \
+                + "6 weeks later a repeat CXR was performed- a few solid dense findings in the right lung. " \
+                + "Therefore, she was referred for a PET-CT which demonstrated increased uptake in the right " \
+                + "breast, lymph nodes on the right a few areas in the lungs and liver. " \
+                + "On biopsy from the lesion in the right breast- triple negative adenocarcinoma. Genetic " \
+                + "testing has not been done thus far. " \
+                + "Genetic counseling- the patient denies a family history of breast, ovary, uterus, " \
+                + "and prostate cancer. Her mother has chronic lymphocytic leukemia (CLL). " \
+                + "She is planned to undergo genetic tests because the aggressive course of the disease, " \
+                + "and her young age. " \
+                + "Impression:" \
+                + "Stage 4 triple negative breast adenocarcinoma. " \
+                + "Could benefit from biological therapy. " \
+                + "Different treatment options were explained- the patient wants to get a second opinion."
+
+patient_document2 = PatientDocument(type=DocumentType.NOTE,
+                                    id="doc2",
+                                    content=DocumentContent(source_type=DocumentContentSourceType.INLINE,
+                                                            value=doc_content2),
+                                    clinical_type=ClinicalDocumentType.PATHOLOGY,
+                                    language="en",
+                                    created_date_time=datetime.datetime(2021, 10, 20))
+
+doc_content3 = "PATHOLOGY REPORT" \
```

Comment:
In python, you can use `"""` for multi-lines string

Reply:
thanks for that.
fixed